src/BSON/Binary.c

```diff
@@ -92,7 +93,7 @@ static HashTable* php_phongo_binary_get_properties_hash(phongo_compat_object_han
 
 /* {{{ proto void MongoDB\BSON\Binary::__construct(string $data, int $type)
    Construct a new BSON binary type */
-static PHP_METHOD(Binary, __construct)
+PHP_METHOD(MongoDB_BSON_Binary, __construct)
```

Comment:
Since the generated arginfo files declares these methods as non-static, we have to remove this from all classes. @jmikola is there a particular reason these were declared as static?

Reply:
It was just a matter of avoiding namespace pollution when compiling. These functions aren't referenced outside of the corresponding C file.

Reply:
Ack, in that case we'll have to live with the polluted namespace, since we can't change the generated stub files :(

Reply:
I realised that using `generate-function-entries static` in the stub file fixes this, so we're back to `static PHP_METHOD(...)` :)